css/style.css

```diff
@@ -78,12 +80,19 @@ button {
 .subtopic {
   text-align: center;
 }
+
+/* styling footer */
 footer {
   text-align: center;
 }
-footer a {
-  height: 2rem;
-  padding: 0.3rem;
+h6 {
+  font-weight: lighter;
+}
+.icon {
```

Comment:
It is best practice to wrap` <img> `tags in divs. Then you apply border-radius not to the actual image but to that div. This will help avoid corners of the images being cropped but you will have circles around.
```
<div class="social">
    <img src="img/instagram-icon.svg" alt="Instagram" class="social_icon"/>
</div>
```
```
.social {
  /* each social_icon wrapped in <div> to create circle border without cutting an image */
  height: 20px;
  width: 20px;
  padding: 8px;
  border-radius: 50px;
  border: solid #dddddd 1px;
  margin: 10px;
}

.social_icon {
  height: 20px;
  width: 20px;
}
```

Reply:
Dear Margareta, my mentor mentioned that while reviewing and I thought I did that.  I don't know why it didn't show up. Thanks again for reminding me. will do it again.